src/core/annotation.js

```diff
@@ -900,6 +902,61 @@ class TextAnnotation extends Annotation {
   }
 }
 
+class FreeTextAnnotation extends Annotation {
+  constructor(parameters) {
+    super(parameters);
+
+    this.data.annotationType = AnnotationType.FREETEXT;
+    let dict = parameters.dict;
+    this.pdfManager = parameters.pdfManager;
+
+    this.data.textColor = createRgbColor(dict.getArray('TextColor'));
+    this.data.textStyle = dict.get('DS');
+    this.data.fontSize = dict.get('FontSize');
+    this.data.opacity = dict.get('CA');
+    this.data.defaultAppearance = dict.get('DA');
+
+    this._preparePopup(dict);
+  }
+
+  getOperatorList(evaluator, task, renderForms) {
+    if (renderForms || this.appearance) {
+      return super.getOperatorList(evaluator, task, renderForms);
+    }
+
+    let data = this.data;
+
+    let opList = new OperatorList();
+    let appearanceStream = new Stream(stringToBytes(data.defaultAppearance));
+    let resourcesFonts = this.pdfManager.pdfDocument.catalog.fontCache;
+
+    return evaluator.getOperatorList({
+      stream: appearanceStream,
+      task,
+      resources: resourcesFonts,
+      operatorList: opList,
+    }).then(() => {
+      let a = opList.argsArray;
+      let i;
+      for (i = 0; i < opList.fnArray.length; i++) {
+        let fn = opList.fnArray[i];
+        switch (fn | 0) {
+          case OPS.setFont:
+            data.fontRefName = a[i][0];
+            data.fontSize = a[i][1];
+            break;
+          case OPS.setGrayFill:
```

Comment:
`setGrayFill` doesn't seem to exist, while `setFillGray` does (https://github.com/mozilla/pdf.js/blob/9c764da8af99a2e896e151108d74a5660f5e9332/src/shared/util.js#L231). Note that they have a different number of arguments (https://github.com/mozilla/pdf.js/blob/b40fb3814a05e7a9c489d995d46b3239e5936d29/src/core/evaluator.js#L2892-L2894) so please check if this works well. If you have not found a case of `setFillGray` in your test documents, let's just remove it from this patch.